weighted_levenshtein/clev.pyx

```diff
@@ -1,22 +1,12 @@
 #!python
-# cython: language_level=2, boundscheck=False, wraparound=False, embedsignature=True, linetrace=True
+# cython: language_level=2, boundscheck=False, wraparound=False, embedsignature=True, linetrace=True, c_string_type=str, c_string_encoding=ascii
 # distutils: define_macros=CYTHON_TRACE_NOGIL=1
 
 from libc.stdlib cimport malloc, free
 
 from clev cimport DTYPE_t, DTYPE_MAX, ALPHABET_SIZE
 
-
-cdef size_t ii, jj
-cdef DTYPE_t unit_array[ALPHABET_SIZE]
-cdef DTYPE_t unit_matrix[ALPHABET_SIZE][ALPHABET_SIZE]
-
-for ii in range(ALPHABET_SIZE):
-    unit_array[ii] = 1
-
-for ii in range(ALPHABET_SIZE):
-    for jj in range(ALPHABET_SIZE):
-        unit_matrix[ii][jj] = 1
+import numpy as np 
```

Comment:
Prior to this change, `numpy` was only a requirement for the package to run tests, but not actually to use any of the functions in the `weighted-levenshtein` package. Can we still use the custom-defined unit array and unit matrix for Python 3 to prevent adding `numpy` as a requirement for the package?